src/Libraries/Microsoft.Extensions.AI.Evaluation/IEvaluator.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.Extensions.AI.Evaluation;
 /// <summary>
 /// Evaluates responses produced by an AI model.
 /// </summary>
+/// <related type="Article" href="/dotnet/ai/tutorials/evaluate-with-reporting">Tutorial: Evaluate a model's response with response caching and reporting</related>
```

Comment:
Did you intend to use the other link here (i.e. evaluate-ai-response). IEvaluator does not depend on Reporting concepts -  so maybe better to link the other article here.

Reply:
I chose the tutorial since it includes a custom evaluator that implements IEvaluator, but I'll change it to the quickstart.